app/k9mail/src/main/java/com/fsck/k9/widget/unread/UnreadWidgetDataProvider.kt

```diff
@@ -23,12 +23,10 @@ class UnreadWidgetDataProvider(
     private val folderNameFormatterFactory: FolderNameFormatterFactory
 ) {
     fun loadUnreadWidgetData(configuration: UnreadWidgetConfiguration): UnreadWidgetData? = with(configuration) {
-        if (SearchAccount.UNIFIED_INBOX == accountUuid) {
-            loadSearchAccountData(configuration)
-        } else if (folderId != null) {
-            loadFolderData(configuration)
-        } else {
-            loadAccountData(configuration)
+        when {
```

Comment:
Can you please revert this change. I find it makes the code harder to read than the old version.

Reply:
:heavy_check_mark: Changed in 117af496994763ef11c5265777cc4cef3dcdce43.